app/views/research_sessions/focus.html.erb

```diff
@@ -1,17 +1,17 @@
 <%= render 'back_link' %>
 <div class="main-content">
   <%= render 'error_summary' %>
-  <h1 class="visually-hidden">Research Focus</h2>
```

Comment:
Somehow this has resulted in 
![hint issue](https://user-images.githubusercontent.com/56056/29018615-ad43d496-7b53-11e7-97d2-bfd2817d101b.PNG)